uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Hook/PageRun/Resources.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class CRM_Tasksassignments_Hook_PageRun_Resources {
+
+  private $resources;
+
+  /**
+   * @param CRM_Core_Resources $resources
+   */
+  public function __construct(CRM_Core_Resources $resources) {
+    $this->resources = $resources;
+  }
+
+  /**
+   * Adds resources needed by specific Task and Workflow pages.
+   *
+   * @param CRM_Core_Page $page
+   */
+  public function handle($page) {
+    if (!$this->shouldHandle($page)) {
+      return;
+    }
+
+    $this->resources->addScriptFile('uk.co.compucorp.civicrm.tasksassignments',
```

Comment:
I see you use `uk.co.compucorp.civicrm.tasksassignments` a few times in this file, how about making it a class constant? In newer extensions CiviCRM provides a class for doing this, but I don't think T&A has an up-to-date civix.php file so it might not be an option here, or you could look into updating this file to match current standards